e2e/tests/nightly/admin/lock-workspace.spec.ts

```diff
@@ -65,8 +66,8 @@ describe('Workspace Admin lock-workspace', () => {
     const aboutLockedIcon = aboutPage.getAboutLockedWorkspaceIcon();
     expect(aboutLockedIcon).toBeTruthy();
     // verify DATA & ANALYSIS tabs are disabled
-    await aboutPage.openDataPage({ waitPageChange: false });
-    await aboutPage.openAnalysisPage({ waitPageChange: false });
+    // await aboutPage.openDataPage({ waitPageChange: false });
```

Comment:
why this change?

Reply:
Because of the changes that happened in this file: [workspace-base.ts](https://github.com/all-of-us/workbench/pull/6276/files#diff-f0d5c7b7e18c9b2a759b0324fb48ae1be874e4e02bcdd4dd58e8a9eb38deaff1)